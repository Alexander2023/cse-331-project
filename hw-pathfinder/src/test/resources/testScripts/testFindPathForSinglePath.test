# Tests that a path is found when there is only one possible path

CreateGraph g

AddNode g p1
AddNode g p2
AddNode g p3

AddEdge g p1 p2 1.0
AddEdge g p2 p3 1.0

FindPath g p1 p3