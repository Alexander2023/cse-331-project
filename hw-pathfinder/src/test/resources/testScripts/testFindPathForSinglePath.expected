# Tests that a path is found when there is only one possible path

created graph g

added node p1 to g
added node p2 to g
added node p3 to g

added edge 1.000 from p1 to p2 in g
added edge 1.000 from p2 to p3 in g

path from p1 to p3:
p1 to p2 with weight 1.000
p2 to p3 with weight 1.000
total cost: 2.000